application/libraries/Omeka/Storage/Adapter/ZendS3.php

```diff
@@ -63,6 +64,10 @@ public function __construct(array $options = array())
         if (!empty($options[self::ENDPOINT_OPTION])) {
             $this->_s3->setEndpoint($options[self::ENDPOINT_OPTION]);
         }
+        
+        if (isset($this->_options[self::FORCE_SSL]) && parse_url($this->_s3->getEndpoint(), PHP_URL_SCHEME)=='http'){
```

Comment:
I was puzzling over the "any value" in the documentation and then noticed this... I was thinking of isset more in the way of a "guard" for checking the actual value, but I think we should still be checking that it's true rather than simply if it's set.